flink-ml-core/src/main/resources/META_INF/NOTICE

```diff
@@ -1,10 +1,10 @@
-flink-ml-uber
+flink-ml-core
```

Comment:
Why do we change it? I think this should refer to the module name (i.e. `flink-ml-uber`).

Reply:
Since blas is not used by `flink-ml-uber`, but `flink-ml-core`. BTW, the new `NOTICE` file is under the directory of `flink-ml-core`

Reply:
Does flink-ml-core need to have NOTICE, if its jar does not have any external classs?

Reply:
Thanks for pointing this out. Now it is moved back to `flink-ml-uber` now.